z3/src/z3_term.cpp

```diff
@@ -104,26 +104,159 @@ bool Z3Term::compare(const Term & absterm) const
 
 Op Z3Term::get_op() const
 {
-  throw NotImplementedException("get_op not yet implemented.");
+  if (is_function)
+  {
+    throw IncorrectUsageException("A function has no operator.");
```

Comment:
Here you can just return a null operator: `return Op()`. In smt-switch, you can still get the operator, it's just null.